guides/plugins/plugins/content/stock/loading-stock-information-from-different-source.md

```diff
@@ -0,0 +1,139 @@
+# Loading Stock Information from a Different Source
+
+## Overview
+
+If Shopware is not the source of truth for your stock data, you can customize the stock loading process and provide your data from a third-party source.
+
+## Prerequisites
+
+We will be decorating a service; therefore, familiarising yourself with the [Adjusting a Service](../../plugin-fundamentals/adjusting-service) guide will be helpful.
```

Comment:
```suggestion
Here again, you will be decorating a service; therefore, it will be helpful to familiarise yourself with the [Adjusting a Service](../../../../../plugin-fundamentals/adjusting-service) guide.
```